### DISCUSSION TITLE
Modify comment field - remove a string

### DISCUSSION THREAD

**Discussion by arsaboo** [beetbox/beets#4768] [Q&A] [answered]
Some of my tracks have strings such as `© 2023 Zee Music Company, encoded using Handbrake`. I would like to remove the `, encoded using Handbrake` portion so that the resulting `comments` would be `© 2023 Zee Music Company`. 

Any guidance on how to construct the appropriate `beet modify` command?

**Comment by sampsyo** [comment, marked as answer]
Unfortunately, there's no built-in way to do complex string manipulations like that. One creative (although complicated) solution might be to use the new "formatted modify" functionality (#488) together with the `inline` plugin. That is, you could define an inline field like `$truncated_comments` to be `comments.split(',', 1)[0]` or similar and then do `beet modify comments=$truncated_comments`. Could work!!

**Reply by arsaboo** [reply]
Thanks @sampsyo that works 🎉
